tests/test_enterprise/api/test_views.py

```diff
@@ -216,16 +217,16 @@ def test_user_view(self, factory, url, sorting_key, model_items, expected_json):
         ),
         (
             factories.EnterpriseCustomerCatalogFactory,
-            reverse('enterprise-catalog-list'),
+            reverse('enterprise-customer-catalog-list'),
```

Comment:
@UmanShahzad It might make sense to introduce a constant for this as well. That way, we'd have all relevant endpoints listed at the top. Naming them might be a bit awkward, but mentioning all relevant endpoints at the top of the file would make the duplication that we currently have (and want to get rid of) more visible.